src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+function sortList() {
+  const list = [...document.querySelectorAll('li')];
+
+  list.sort((a, b) => {
+    const item1 = parseInt(a.getAttribute('data-salary').replace(/[\D]/g, ''));
+    const item2 = parseInt(b.getAttribute('data-salary').replace(/[\D]/g, ''));
+
+    return item2 - item1;
+  });
+
+  list.forEach((item) => {
+    document.querySelector('ul').appendChild(item);
+  });
+}
+
+function getEmployees() {
+  const list = [...document.querySelectorAll('ul')];
```

Comment:
In the `getEmployees` function, you are selecting all 'ul' elements. This is incorrect as you should be selecting 'li' elements instead. Please correct this.